auth-foundation-bootstrap/src/test/java/com/okta/authfoundationbootstrap/CredentialBootstrapTest.kt

```diff
@@ -103,4 +103,9 @@ class CredentialBootstrapTest {
         assertThat(credential1.await()).isSameInstanceAs(credential2.await())
         assertThat(CredentialBootstrap.credentialDataSource.listCredentials()).hasSize(1)
     }
+
+    @Test fun testOidcClient() {
```

Comment:
Nit: Given what the test is doing, maybe rename it to testOidcClientInitialization?